ibm/data_source_ibm_is_volume.go

```diff
@@ -237,6 +237,11 @@ func volumeGet(d *schema.ResourceData, meta interface{}, name string) error {
 	if err != nil {
 		return err
 	}
+	listVolumesOptions := &vpcv1.ListVolumesOptions{}
+	if v, ok := d.GetOk(isVolumeName); ok {
```

Comment:
I don't understand why are you creating option here already in for loop we have it